test/test_pub_sub.py.in

```diff
@@ -181,7 +181,7 @@ def generate_test_description(ready_fn):
     node_main_test = Node(
         package='performance_test', node_executable='perf_test', output='log',
         arguments=[
-            '-c', '@COMM@',
+            '-c', 'ROS2',
             '-t', '@PERF_TEST_TOPIC@',
```

Comment:
We need to pick a value for this. Right now, it's inheriting whatever topic was last iterated from the list of topics. We should pick a single topic so that the comparisons are fair.

Right now it's `Array2m`, but that will be broken on older ROS 2 distros with Fast-RTPS sync. I suggest `Array60k`, but I'd like your thoughts on what to do here.

Reply:
Originally this value it was Array1k but I'm also fine with Array60k.

Reply:
118c9ff